src/fetch_app_status.rb

```diff
@@ -68,6 +72,18 @@ def get_version_info(app)
   }
 end
 
+def get_build_info(app)
+  builds = app.get_builds.sort_by(&:uploaded_date).reverse[0, number_of_builds]
+
+  builds.map do |build|
+    {
+      "version" => build.version,
+      "uploaded_data" => build.uploaded_date,
+      "status" => build.processing_state,
```

Comment:
Can we switch to more modern syntax?

```suggestion
      version: build.version,
      uploaded_data: build.uploaded_date,
      status: build.processing_state,
```